recipes/emsarray/meta.yaml

```
@@ -0,0 +1,76 @@
+{% set name = "emsarray" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/emsarray-{{ version }}.tar.gz
+  sha256: 63c92c95483ec263aecf88266acf102513e2ce74099c035172affe4eff959729
+
+build:
+  number: 0
+
+outputs:
+  - name: emsarray-core
+    build:
+      noarch: python
+      script: python -m pip install . -vv --no-deps
+      entry_points:
+        - emsarray = emsarray.cli:main
+
+    requirements:
+      host:
+        - python >=3.8
+        - pip
+      run:
+        - python >=3.8
+        - bottleneck >=1.3
+        - geojson >=2.5.0
+        - netcdf4 >=1.5.3
+        - numpy >=1.18.0
+        - shapely >=1.8.0
+        - xarray >=0.18.2
+    test:
+      imports:
+        - emsarray
+      commands:
+        - pip check
+        - emsarray --help
+      requires:
+        - pip
+
+  - name: emsarray
+    build:
+      noarch: python
```

```suggestion
      noarch: generic
      string: {{ build_num }}_fullpackage
```
Generic since you're not actually building anything and string so that it is clearer what this is.